bin/config.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Maybe `config-example.json` as we don't really use it?

Reply:
This config can be used to test `metatests` itself

Reply:
It can but it is not used therefore it's an example, not?

Reply:
I can change `npm test` to use `./bin/cli.js -c ./bin/config.json`. WDYT?

Reply:
Thinking about it, running tests only in Node environment doesn't require config file, so I'll rename it. But what about `npm test`? Should it use cli or just leave as it is?

Reply:
Yeah, change it to cli and remove `test/all.js` file as it's not needed anymore.
Would `./bin/cli.js test` be possible?

Reply:
`./bin/cli.js` will work same as `./bin/cli.js test`, but do we need `test/speed.js`? Because it's excluded from `test/all.js`.

Reply:
Oh, I actually wanted to remove that as it is not a test. But that's a separate PR. It's fine with it for now.

Reply:
Well, I added it only due to your suggestion https://github.com/metarhia/metatests/pull/75#discussion_r214552387

Reply:
Well, I had it since https://github.com/metarhia/metatests/pull/75#discussion_r214552387. I also don't like current implementation, but I think it would be nice to have default option. Either way we have two options:
1. remove it at all
2. show message that no test files were provided and it will try to load files from test folder

There is a point in forcing user to explicitly declare test files, but I don't see much sense in the first option, because why would you use testing cli and not provide any files?
In the end, I like second option, but taking to consideration that other people in their projects may have whole different test files location and patterns, first one sounds more reasonable

Reply:
In that comment I didn't consider that as a default folder, I just pointed out constant extraction.

I think just removing it and showing user `help` if no files (or config) were provided is a good (and consistent with other cli tools, at least in linux) option. Similar to other places where we removed defaults and just fail there.